Assets/Scripts/GhostController.cs

```diff
@@ -32,11 +36,20 @@ public class GhostController : MonoBehaviour {
 			transform.localScale = flipXScale;
 		}
 		if(Input.GetKey(KeyCode.Space)) {
+            //FIXME: do not keep the force increasing
```

Comment:
You can check whether the GameObject is on the ground, if it is, add force, if not, not add